lib/Serge/Command/show.pm

```diff
@@ -0,0 +1,34 @@
+package Serge::Command::show;
+use parent Serge::Command;
+
+use strict;
+
+use Config::Neat::Inheritable;
+use Config::Neat::Render;
+
+sub get_commands {
+    return {
+        show => {
+            handler => \&run,
+            info => 'Show expanded version of a configuration file',
+            need_config => 1,
+        },
+    }
+}
+
+sub run {
+    my ($self) = @_;
+
+    my @config_files = $self->{parent}->get_config_files;
+    die "Multiple configuration files not allowed\n" unless $#config_files == 0;
```

Comment:
I wasn't sure if you would prefer to use `$#config_files == 0` or rather `scalar @config_files == 1`. Please let me know if you have any preferences.